scripts/publish/publish.ps1

```diff
@@ -135,6 +135,17 @@ function UploadInstallers($msiFile)
     return 0
 }
 
+function UploadVersionBadge($badgeFile)
+{
+    $fileName = [System.IO.Path]::GetFileName($badgeFile)
+    
+    Write-Host "Uploading the version badge to Latest"
+    UploadFile "dev/Binaries/Latest/$filename" $badgeFile
+    
+    Write-Host "Uploading the version badge to $env:DOTNET_CLI_VERSION"
+    UploadFile "dev/Binaries/$env:DOTNET_CLI_VERSION/$filename" $badgeFile
```

Comment:
Yes if you import _common.dh in your scripts. In PS you can just hardcode it to "windows"?